src/py/flwr/cli/new/templates/app/README.baseline.md.tpl

```diff
@@ -0,0 +1,128 @@
+---
+title: title of the paper
+url: https://arxiv.org/abs/2007.14390 # update with the link to your paper
+labels: [label1, label2] # please add between 4 and 10 single-word (maybe two-words) labels (e.g. system heterogeneity, image classification, asynchronous, weight sharing, cross-silo). Do not use "". Remove this comment once you are done.
+dataset: [dataset1, dataset2] # list of datasets you include in your baseline. Do not use "". Remove this comment once you are done.
```

Comment:
Should we add a `TODO` at the beginning of each of those comments? We can then easily search if any TODOs are left